cmd/gapic-showcase/run.go

```diff
@@ -53,8 +53,15 @@ func init() {
 			}
 			s := grpc.NewServer(opts...)
 			defer s.GracefulStop()
+
+			// Register Services to the server.
```

Comment:
do you really want this commit in this PR? Shouldn't this branch contain only the version changes?

Reply:
I'll rebase after #64 is merged and this will disappear from this PR.